Source/WebKit/UIProcess/API/C/WKAPICast.h

```diff
@@ -419,6 +423,114 @@ inline WKStorageBlockingPolicy toAPI(WebCore::StorageBlockingPolicy policy)
     return kWKAllowAllStorage;
 }
 
+inline OptionSet<WebsiteDataType> toWebsiteDataTypes(WKWebsiteDataTypes websiteDataTypes)
```

Comment:
Let's put the body of this in a cpp file.  We don't need to include it everywhere, and one function call isn't expensive compared to the body of the function.

Reply:
I'll fix it.

Reply:
I have a question. Does this mean creating WKAPICast.cpp? Or do you mean defining it as a static function in WKWebsiteDataStoreRef.cpp and WKWebsiteDataRecordRef.cpp?